corehq/apps/smsforms/migrations/0003_auto__chg_field_sqlxformssession_couch_id.py

```diff
@@ -0,0 +1,43 @@
+# encoding: utf-8
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Changing field 'SQLXFormsSession.couch_id'
+        db.alter_column(u'smsforms_sqlxformssession', 'couch_id', self.gf('django.db.models.fields.CharField')(max_length=50))
```

Comment:
I don't think upgrading will make a difference. I found this in the [latest docs](http://www.postgresql.org/docs/9.4/static/sql-altertable.html): "Adding a CHECK or NOT NULL constraint requires scanning the table to verify that existing rows meet the constraint."

Also http://dba.stackexchange.com/questions/66840/add-not-null-constraint-to-large-table-without-table-scan